<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton:hover{
	background-color: rgb(204, 255, 137);
border:none;
}
QWidget{
border:none;
	

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>400</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="curLabel">
           <property name="styleSheet">
            <string notr="true">QLabel{
	font: 22pt &quot;Microsoft YaHei UI&quot;;
	
	color: rgb(34, 85, 31);
	
}
</string>
           </property>
           <property name="text">
            <string>00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="toalLabel">
           <property name="styleSheet">
            <string notr="true">QLabel{
	font: 22pt &quot;Microsoft YaHei UI&quot;;
	
	color: rgb(34, 85, 31);
	
}
</string>
           </property>
           <property name="text">
            <string>00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>400</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="MusicSlider">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="ChooseMusic">
           <property name="styleSheet">
            <string notr="true">*{
border:none;


}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="piclibs.qrc">
             <normaloff>:/pic/C:/Users/asus/Pictures/rc/chat/file.png</normaloff>:/pic/C:/Users/asus/Pictures/rc/chat/file.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="ListMusic">
           <property name="styleSheet">
            <string notr="true">*{
border:none;


}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/asus/Pictures/rc/list.png</normaloff>C:/Users/asus/Pictures/rc/list.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="preMusic">
           <property name="styleSheet">
            <string notr="true">*{
border:none;


}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/asus/Pictures/rc/previous.png</normaloff>C:/Users/asus/Pictures/rc/previous.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="beginMusic">
           <property name="styleSheet">
            <string notr="true">*{
border:none;


}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/asus/Pictures/rc/pause.png</normaloff>C:/Users/asus/Pictures/rc/pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="nextMusic">
           <property name="styleSheet">
            <string notr="true">*{
border:none;


}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/asus/Pictures/rc/next.png</normaloff>C:/Users/asus/Pictures/rc/next.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="voice">
           <property name="styleSheet">
            <string notr="true">*{
border:none;


}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/asus/Pictures/rc/voice.png</normaloff>C:/Users/asus/Pictures/rc/voice.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="piclibs.qrc"/>
 </resources>
 <connections/>
</ui>
